src/mmw/js/src/modeling/controllers.js

```diff
@@ -38,7 +38,8 @@ var ModelingController = {
                 scenarios: new models.ScenariosCollection([
                     new models.ScenarioModel({
                         name: 'Current Conditions',
-                        is_current_conditions: true
+                        is_current_conditions: true,
+                        active: true
```

Comment:
I think there's already some discussion about the active scenario tab going on in some work by @caseypt and @kdeloach